xmlrunner/result.py

```diff
@@ -64,7 +66,7 @@ def to_unicode(data):
         # Try utf8
         return six.text_type(data)
     except UnicodeDecodeError:
-        return repr(data).decode('utf8', 'replace')
+        return str(data).decode('utf8', 'replace')
```

Comment:
I thought this code never runs on python 3 because of 
```
     if six.PY3:
         return six.text_type(data)
```

I dont know how to provide compatibility with python versions here. I can remove the commit to leave this line for future) should I do that?